launcher/ui/CMakeLists.txt

```diff
@@ -44,6 +44,20 @@ target_link_libraries(gammaray_launcher_ui
 
 install(TARGETS gammaray_launcher_ui EXPORT GammaRayTargets ${INSTALL_TARGETS_DEFAULT_ARGS})
 
+gammaray_install_headers(DESTINATION launcher
+    ${CMAKE_CURRENT_BINARY_DIR}/gammaray_launcher_ui_export.h
+    attachdialog.h
+    connectpage.h
+    launcherwindow.h
+    launchpage.h
+    probeabimodel.h
+    processfiltermodel.h
+    processlist.h
+    processmodel.h
+    promolabel.h
+    selftestpage.h
```

Comment:
Do we really need all of them installed? Some of these aren't even exported.

Anyway, feel free to push this with the files you need.